### PR TITLE
chore(showcase): New submissions for site showcase

### PR DESCRIPTION
**Pull request by flowen**
## Description

2 submissions for the site showcase

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -7910,3 +7911,40 @@
     - Web Development
   built_by: IBM
   featured: true
+- title: Mevish Aslam, business coach
+  url: "https://mevishaslam.com/"
+  main_url: "https://mevishaslam.com/"
+  source_url: ""
```
Comment (pieh) [inline docs/sites.yml]: Please remove this field entirely if you don't want to provide url to source code / project is private

File: docs/sites.yml [hunk 2]
```diff
@@ -7910,3 +7911,40 @@
...
+    - Freelance
+    - Marketing
+    - Portfolio
+  built_by: Rou Hun Fan
+  built_by_url: "https://flowen.me"
+  featured: false
+- title: Principles of wealth
+  url: "https://principlesofwealth.net"
+  main_url: "https://principlesofwealth.net"
+  source_url: ""
```
Comment (pieh) [inline docs/sites.yml]: Same as above

File: docs/sites.yml (hunk 1 above)
Comment (flowen) [reply docs/sites.yml]: I see, thanks for checking!

File: docs/sites.yml (hunk 2 above)
Comment (flowen) [reply docs/sites.yml]: should be good now

**Pull request merged**